src/Apps/MyCollection/Routes/EditArtwork/MyCollectionArtworkForm.tsx

```diff
@@ -130,6 +131,10 @@ export const MyCollectionArtworkForm: React.FC<MyCollectionArtworkFormProps> = (
         })
       )
 
+      // Waiting for a few seconds to make sure the new images are processed
+      // and ready to be displayed
+      await wait(5000)
```

Comment:
**issue:** I feel this is a lot to be added to the waiting time in addition to the time it will take to upload things already.
We might as well need to clear the timeout

Reply:
I've adjusted the time to 3 seconds, which works in most cases on Staging. Let's test this on production.

Also, the timeout is only added now when the user has added new images to the artwork.

Reply:
now since artsy/metaphysics#4354 is merged, do we still need to wait?

Reply:
Unfortunately, I think we need to. I've tested it on Staging and if we navigate the user right away, there might be no version of the image at all that has been processed.

Happy to hear a better or cleaner solution. We could try to include `me` and `artwork` in the response of the mutation and update the Relay store like this. It could be a great Future Friday project to try make it work :)

Reply:
Yeah, I think it's a good idea to explore a cleaner solution, but maybe for the time being, I think we should merge this temporary solution to make the experience a bit better than the current one. @MounirDhahri any thoughts?